bitcoin_transfer/spend_your_coin.md

```diff
@@ -83,60 +83,65 @@ Do you remember the main questions? **From where, to where and how much?**
-If you succeed in completing this challenge you will be able to find your contribution amongst the **Hall of the Makers** on [http://n.bitcoin.ninja/](http://n.bitcoin.ninja/) \(ordered by generosity\).
+> The donation address of this book is: [1KF8kUVHK42XzgcmJF4Lxz4wcL5WDL97PB](https://blockchain.info/address/1KF8kUVHK42XzgcmJF4Lxz4wcL5WDL97PB)  
+This money goes into Nicolas' "Coffee and Sushi Wallet" that will keep him fed and compliant while writing the rest of the book.  
+If you succeed in completing this challenge on the MainNet you will be able to find your contribution among the **Hall of the Makers** on [http://n.bitcoin.ninja/](http://n.bitcoin.ninja/) \(ordered by generosity\).
 
+To get the our MainNet address:
 ```cs
 var hallOfTheMakersAddress = new BitcoinPubKeyAddress("1KF8kUVHK42XzgcmJF4Lxz4wcL5WDL97PB");
 ```
 
-If you are working on the testnet, send the testnet coins to any testnet address.
+Or if you are working on the TestNet, send the TestNet coins to [our TestNet address](https://live.blockcypher.com/btc-testnet/address/mzp4No5cmCXjZUpf112B1XWsvWBfws5bbB/).
 
 ```cs
-var hallOfTheMakersAddress = BitcoinAddress.Create("mzp4No5cmCXjZUpf112B1XWsvWBfws5bbB");
+var hallOfTheMakersAddress = BitcoinAddress.Create("mzp4No5cmCXjZUpf112B1XWsvWBfws5bbB", Network.TestNet);
 ```
 
 ### How much?
 
-If you want to send **0.5 BTC** from a **transaction input** with **1 BTC** you actually have to spend it all!  
-As the diagram shows below, your **transaction output** specifies  **0.5** BTC to Hall of The Makers and **0.4999** back to you.  
-What happens to the remaining **0.0001 BTC**? This is the miner fee in order to incentivize them to add this transaction into their next block.
+Bitcoin has [several units to use](https://en.bitcoin.it/wiki/Units). 1 bitcoin (BTC) is 1'000 milli-bitcoins/millibits. 1 milli-bitcoin (mBTC) is 1'000 micro-bitcoins/bits. And finally 1 (&micro;BTC) micro-bitcoin is 100 satoshis. 1 satoshi (sat) is the smallest unit on the Bitcoin network.  
+
+If you want to send **400 &micro;BTC** (a few dollars) from an **unspent output**, which holds **1'000 &micro;BTC**, you actually have to spend it all!  
+As the diagram shows below, your **transaction output** specifies  **400 &micro;BTC** to [Hall of The Makers](http://n.bitcoin.ninja/) and **530 &micro;BTC** back to you.  
+What happens to the remaining **70 &micro;BTC**? This is the _miner fee_.  
+The miner fee incentivizes the miners to add this transaction into their next block. The higher the miner fee the more motivated the miner is to include your transaction in the next block, meaning that your transaction will be confirmed faster. If you set the miner fee to zero, your transaction might never be confirmed.
 
 ![](../assets/SpendTx.png)
 
 ```cs
 TxOut hallOfTheMakersTxOut = new TxOut()
 {
-    Value = new Money((decimal)0.5, MoneyUnit.BTC),
+    Value = new Money(400, MoneyUnit.Bit),
```

Comment:
No, please:) Btw `0.5m` would be better, than the original one.

Reply:
Ok, I think it too that 0.5m is better than the original, it was not consistent before.

So, if we decide to not use the MoneyUnit.Bit, I change all of them to .BTC. @NicolasDorier?